src/transmuxer-worker.js

```diff
@@ -8,35 +8,27 @@
  * transmuxer running inside of a WebWorker by exposing a simple
  * message-based interface to a Transmuxer object.
  */
-var
-  muxjs = {},
-  transmuxer,
-  initOptions = {};
-
-importScripts('../node_modules/mux.js/lib/utils/stream.js');
-importScripts('../node_modules/mux.js/lib/utils/exp-golomb.js');
-importScripts('../node_modules/mux.js/lib/mp4/mp4-generator.js');
-importScripts('../node_modules/mux.js/lib/codecs/aac.js');
-importScripts('../node_modules/mux.js/lib/codecs/h264.js');
-importScripts('../node_modules/mux.js/lib/m2ts/m2ts.js');
-importScripts('../node_modules/mux.js/lib/m2ts/caption-stream.js');
-importScripts('../node_modules/mux.js/lib/m2ts/metadata-stream.js');
-importScripts('../node_modules/mux.js/lib/mp4/transmuxer.js');
+let muxjs = require('mux.js');
+let gTransmuxer;
+let initOptions = {};
 
 /**
  * wireTransmuxerEvents
  * Re-emits tranmsuxer events by converting them into messages to the
  * world outside the worker
  */
-var wireTransmuxerEvents = function (transmuxer) {
-  transmuxer.on('data', function (segment) {
-    // transfer ownership of the underlying ArrayBuffer instead of doing a copy to save memory
+const wireTransmuxerEvents = function(transmuxer) {
+  transmuxer.on('data', function(segment) {
+    // transfer ownership of the underlying ArrayBuffer
+    // instead of doing a copy to save memory
     // ArrayBuffers are transferable but generic TypedArrays are not
+    /* eslint-disable max-len */
     // see https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#Passing_data_by_transferring_ownership_(transferable_objects)
+    /* eslint-enable max-len */
```

Comment:
We could use a shortened URL to get around the line length problem: https://goo.gl/TBDNny